docs/allow-nested-if-parent-exist.md

```diff
@@ -0,0 +1,51 @@
+# AllowNestedIfParentExist
+
+## Category
+
+ARM Error
+
+## Applies to
+
+ARM OpenAPI(swagger) specs
+
+## Related ARM Guideline Code
+
+- RPC-Get-V1-11
+
+## Description
+
+List calls for nested children under the resource group segment is allowed only if parent resource under the resource group exist.
+
+## How to fix
+
+Nested List call without parent call is invalid, ensure to add Parent Resource get call as below valid example or remove the Nested get call from the paths.
+
+### Invalid example
```

Comment:
Following are the patterns for the invalid examples 

GET /subscriptions/{sub1}/providers/{ProviderName}/resourceType1/{resourceTypeName}/nestedResourceType

GET /subscriptions/{sub1}/providers/{ProviderName}/nestedResourceType

GET /subscriptions/{sub1}/resourcegroups/{rgname}/providers/{ProviderName}/nestedResourceType

Reply:
I left this comment as a pointer. You will need to expand on it and show the invalid combinations of parent and child here